lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -48,6 +100,17 @@
                 </layout.leftNav>
                 <layout.main local-class='Main'>
                     {{#if this.inReview}}
+                        {{#unless draftManager.registrationResponsesIsValid}}
+                            {{#if this.showMobileView}}
```

Comment:
I don't love this construct, but it's even grosser combined. We should consider creating something like `draftManager.hasInvalidResponses` that would make these conditions read better:
```hbs
{{#if (and draftManager.hasInvalidResponses this.showMobileView)}}
```
and for the register button:
```hbs
{{#if (and draftManager.hasInvalidResponses (not @showMobileView))}}
```
Or possibly just created computed for each:
```hbs
{{#if this.showInvalidResponsesMessageInMain}}
```
```hbs
{{#if this.showInvalidResponsesMessageInRight}}
```

Reply:
I suppose the latter is local to the `Register` component so maybe should just be:
```hbs
{{#if this.showInvalidResponsesMessage}}
```